### PR TITLE
doc: Add AWS ECS Troubleshoot Section

### PR DESCRIPTION
**Pull request by i3mSuperMan**
#### Summary

Adding troubleshoot section in AWS ECS deployment.

#### Changes
- Added  AWS ECS troubleshooting section.
- Added UDP gateways connection issue after upgrading to the latest version.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/getting-started/aws/ecs/troubleshooting/_index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: "Troubleshooting AWS ECS Deployment"
+description: ""
+weight: 7
+---
+
+<!--
+TODO: https://github.com/TheThingsNetwork/lorawan-stack/issues/2714
+Move to generic getting started guide once ready.
+-->
```
Comment (KrishnaIyer) [inline doc/content/getting-started/aws/ecs/troubleshooting/_index.md:10]: That issue is closed already. What is this for?

File: doc/content/getting-started/aws/ecs/troubleshooting/_index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+This section contains information to troubleshoot {{% tts %}} ECS deployment.
+
+<!--more-->
+
+### TTS no longer accepting connections from UDP gateways after upgrading to latest version
```
Comment (KrishnaIyer) [inline doc/content/getting-started/aws/ecs/troubleshooting/_index.md:16]: This is a bit vague and makes it seems like new TTS versions are buggy. I'd rephrase this as
```
UDP gateways not connected after an update.
```

File: doc/content/getting-started/aws/ecs/troubleshooting/_index.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  ```
+  tenancy:
+        default-id: ''
+  ```
+
+Apply the configurations and restart the `gs` task.
+
+## Professional Support
+
+Additional paid support for this deployment is offered by The Things Industries. You can contact us either by mailing us at `support@thethingsindustries.com` or by visiting [our support page](https://www.thethingsindustries.com/support/).
```
Comment (KrishnaIyer) [inline doc/content/getting-started/aws/ecs/troubleshooting/_index.md:39]: Please only use the support page. The email should be reserved for customers who have already purchased a support plan.

File: doc/content/getting-started/aws/ecs/troubleshooting/_index.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  ```
+  tenancy:
+        default-id: 'default'
+  ```
+  Unset the `DefaultTenantId` as below
+  
+  ```
+  tenancy:
+        default-id: ''
+  ```
```
Comment (KrishnaIyer) [inline doc/content/getting-started/aws/ecs/troubleshooting/_index.md:33]: Have you seen an instance where this changed due a version update? If yes that should not happen. Please file an internal issue for Michal.

**Review by KrishnaIyer — CHANGES_REQUESTED** [review]

File: doc/content/getting-started/aws/ecs/troubleshooting/_index.md (hunk 1 above)
Comment (i3mSuperMan) [reply doc/content/getting-started/aws/ecs/troubleshooting/_index.md]: It was our mistake. Please ignore this, and we have removed it now.

File: doc/content/getting-started/aws/ecs/troubleshooting/_index.md (hunk 4 above)
Comment (i3mSuperMan) [reply doc/content/getting-started/aws/ecs/troubleshooting/_index.md:26]: Ref: https://github.com/TheThingsIndustries/lorawan-stack-support/issues/523

**Comment by i3mSuperMan** [comment]
Thanks, @KrishnaIyer. We have updated the mentioned changes in the PR.

File: doc/content/getting-started/aws/ecs/troubleshooting/_index.md (hunk 4 above)
Comment (KrishnaIyer) [reply doc/content/getting-started/aws/ecs/troubleshooting/_index.md:26]: Ok but this advice is still incorrect. If this happens, it's a bug in the templates and customers should reach out to us. Since we've seem to have fixed this, this (hopefully) won't happen again so it can be removed.

Comment (i3mSuperMan) [reply doc/content/getting-started/aws/ecs/troubleshooting/_index.md:26]: Okay, @KrishnaIyer.

Do you consider closing this PR?

Comment (KrishnaIyer) [reply doc/content/getting-started/aws/ecs/troubleshooting/_index.md:26]: Yes please do.

**Comment by ymgupta** [comment]
As the issue has been fixed, troubleshooting information wouldn't be required anymore. Closing this PR now.

Ref: https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/484#discussion_r680690998
